tests/core/isr_context_switch/tests/01-run.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 Otto-von-Guericke-Universität Magdeburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+# @author      Marian Buschsieweke <marian.buschsieweke@posteo.net>
+
+import sys
+from testrunner import run
+
+
+# use a larger timeout as context switches make take some time on slower MCUs
+TIMEOUT = 60
+
+
+def testfunc(child):
+    child.expect("Testing 500 context switches triggered from ISR")
```

Comment:
```suggestion
    child.expect(r"Testing 500+ context switches triggered from ISR")
```

otherwise it passes on `adafruit-feather-nrf52840-sense`, but also resets to the bootloader.